src/Beberlei/Bundle/MetricsBundle/DependencyInjection/BeberleiMetricsExtension.php

```diff
@@ -49,6 +49,10 @@ private function createCollector($type, $config)
             'event' => 'console.terminate',
         ));
 
+        if (count($config['tags']) > 0) {
+            $definition->addMethodCall('setTags', array($config['tags']));
```

Comment:
this line can lead to an error if the order of tag items is not strictly respected, right ?